server/routes/updateChain.ts

```diff
@@ -128,8 +131,9 @@ const updateChain = async (
   if (custom_stages) chain.custom_stages = custom_stages;
   if (terms) chain.terms = terms;
   if (snapshot) chain.snapshot = snapshot;
-  if (req.body['featured_topics[]'])
-    chain.featured_topics = req.body['featured_topics[]'];
+
+  // TODO Graham 3/31/22: Will this potentially lead to undesirable effects if toggle
```

Comment:
?

Reply:
It's old code I'm worried about, so not relevant to this PR.